javascript/chubbyjs/app.ts

```diff
@@ -0,0 +1,82 @@
+import ServerRequestFactory from '@chubbyjs/chubbyjs-http-message/dist/Factory/ServerRequestFactory';
+import StreamFactory from '@chubbyjs/chubbyjs-http-message/dist/Factory/StreamFactory';
+import UriFactory from '@chubbyjs/chubbyjs-http-message/dist/Factory/UriFactory';
+import NodeResponseEmitter from '@chubbyjs/chubbyjs-node-psr-http-message-bridge/dist/NodeResponseEmitter';
+import PsrRequestFactory from '@chubbyjs/chubbyjs-node-psr-http-message-bridge/dist/PsrRequestFactory';
+import ResponseInterface from '@chubbyjs/psr-http-message/dist/ResponseInterface';
+import ServerRequestInterface from '@chubbyjs/psr-http-message/dist/ServerRequestInterface';
+import { createServer, IncomingMessage, ServerResponse } from 'http';
+
+const responseFactory = new ResponseFactory();
+
+const app = new Application([
+    new ErrorMiddleware(responseFactory, true),
+    new RouteMatcherMiddleware(
+        new PathToRegexpRouteMatcher(
+            new Routes([
+                Route.get(
+                    '/',
+                    'index',
+                    new CallbackRequestHandler(
+                        (): ResponseInterface => {
+                            const response = responseFactory.createResponse(200);
+                            response.getBody().end('');
+
+                            return response;
+                        },
+                    ),
+                ),
+                Route.get(
+                    '/user/:id',
+                    'user_view',
+                    new CallbackRequestHandler(
+                        (request: ServerRequestInterface): ResponseInterface => {
+                            const response = responseFactory.createResponse(200);
+                            response.getBody().end(request.getAttribute('id'));
+
+                            return response;
+                        },
+                    ),
+                ),
+                Route.post(
+                    '/user',
+                    'user_create',
+                    new CallbackRequestHandler(
+                        (): ResponseInterface => {
+                            const response = responseFactory.createResponse(200);
+                            response.getBody().end('');
+
+                            return response;
+                        },
+                    ),
+                ),
+            ]),
+        ),
+        responseFactory,
+    ),
+]);
+
+const psrRequestFactory = new PsrRequestFactory(
+    new ServerRequestFactory(),
+    new UriFactory(),
+    new StreamFactory(),
+);
+
+const nodeResponseEmitter = new NodeResponseEmitter();
+
+const server = createServer((req: IncomingMessage, res: ServerResponse) => {
+    const serverRequest = psrRequestFactory.create(req);
+    const response = app.handle(serverRequest);
+
+    nodeResponseEmitter.emit(response, res);
+});
+
+server.listen(3000);
```

Comment:
do you listen on `0.0.0.0` ?

Reply:
server.listen(3000, '0.0.0.0') should do the trick

Reply:
https://github.com/the-benchmarker/web-frameworks/pull/4621/commits/2fb2a70a26f1bb9c524d24e8e2bf04949f093997